homeassistant/components/climate/ecobee.py

```diff
@@ -318,8 +329,21 @@ def set_hold_mode(self, hold_mode):
 
     def set_auto_temp_hold(self, heat_temp, cool_temp):
         """Set temperature hold in auto mode."""
-        self.data.ecobee.set_hold_temp(self.thermostat_index, cool_temp,
-                                       heat_temp, self.hold_preference())
+        if cool_temp is not None:
```

Comment:
Why are these changes to ecobee included in this PR?

Reply:
Because ecobee could so far only set `target_temperature_high` and `target_temperature_low` at the same time. I figured that if `SUPPORT_TARGET_TEMPERATURE_HIGH` and `SUPPORT_TARGET_TEMPERATURE_LOW` are two different flags, then it must be possible to set only one of them.

Reply:
Will the high/low target temps still be visible in the frontend graph without those supported features?

Reply:
Honestly: I can't figure out how this would currently be shown in the front end. There is code computing whether it should be shown or not in line 

https://github.com/tinloaf/home-assistant-polymer/blob/8201dd34761dbb8feb219e9c9c3cb58e841dd412/src/more-infos/more-info-climate.html#L272

(still based on the old attribute-based detection), but that is never used. The actual temperature controls in 

https://github.com/tinloaf/home-assistant-polymer/blob/8201dd34761dbb8feb219e9c9c3cb58e841dd412/src/more-infos/more-info-climate.html#L95-L108

also don't use it. The included `ha-climate-control` also doesn't seem to support it.

In short: I did not change anything regarding low / high temperatures, I just added the support flag. It does not seem that the front end currently supports setting these values. It would be a next step to implement that in the front end.

Reply:
We used to have a slider with 2 handles to set low and high temp. It got removed in favor of the big buttons and then we lost that functionality.

Reply:
Which functionality is lost?

Reply:
I think there is currently no way of setting `target_temperature_high` or `target_temperature_low` via the front end. You can of course still set it via service calls.